cost/google/rightsize_vm_recommendations/google_rightsize_vm_recommendations.pt

```diff
@@ -0,0 +1,666 @@
+  EOS
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_recommendations" do
+  validate $ds_vm_cost_mapping do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data.instances }} rows found for Rightsizing VM (Compute Instance) Recommendations"
+    check eq(size(val(data, "instances")), 0)
+    escalate $esc_email
+    escalate $esc_terminate_resources
+    export "instances" do
+      resource_level true
+      field "accountID" do
+        label "Project ID"
+      end
+      field "accountName" do
+        label "Project Name"
+      end
+      field "projectNumber" do
+        label "Project Number"
+      end
+      field "resourceID" do
+        label "Resource ID"
+      end
+      field "resourceName" do
+        label "Resource Name"
+      end
+      field "resourceType" do
+        label "Resource Type"
+      end
+      field "zone" do
+        label "Zone"
+      end
+      field "hostname" do
+        label "Hostname"
+      end
+      field "cpuMaximum" do
+        label "CPU Maximum %"
+      end
+      field "cpuMinimum" do
+        label "CPU Minimum %"
+      end
+      field "cpuAverage" do
+        label "CPU Average %"
+      end
+      field "tags" do
+        label "Tags"
+      end
+      field "description" do
+        label "Description"
+      end
+      field "savings" do
+        label "Estimated Monthly Savings"
+      end
+      field "savingsCurrency" do
+        label "Currency"
+      end
+      field "priority" do
+        label "Priority"
+      end
+      field "recommenderSubtype" do
+        label "Sub Type"
+      end
+      field "state" do
+        label "State"
+      end
+      field "status" do
+        label "Status"
+      end
+      field "id" do
+        label "ID"
+        path "resourceID"
+      end
+      field "service" do
+        label "Service"
+      end
+    end
+  end
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+
+escalation "esc_email" do
+  automatic true
+  label "Send Email"
+  description "Send incident email"
+  email $param_email
+end
+
+escalation "esc_terminate_resources" do
+  automatic contains($param_automatic_action, "Terminate Instances")
+  label "Terminate Instances"
+  description "Approval to terminate all selected instances"
+  run "terminate_resources", data, $param_log_to_cm_audit_entries, rs_optima_host
+end
+
+###############################################################################
+# Cloud Workflow
+###############################################################################
+
+define terminate_resources($data,$param_log_to_cm_audit_entries, $$rs_optima_host) return $all_responses do
```

Comment:
Have you tested the cloud workflow part? Or is this part just taken from the Idle VM Recommender policy?

If it's the latter, maybe we remove the Cloud Workflow part (and related escalation) for this MVP version of the policy

Reply:
Done, it was the latter.